src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -71,37 +61,10 @@
  */
 class FirebaseUserManager {
 
-  static final String CONFIGURATION_NOT_FOUND_ERROR = "configuration-not-found";
-  static final String TENANT_ID_MISMATCH_ERROR = "tenant-id-mismatch";
-  static final String TENANT_NOT_FOUND_ERROR = "tenant-not-found";
   static final String USER_NOT_FOUND_ERROR = "user-not-found";
   static final String INTERNAL_ERROR = "internal-error";
```

Comment:
Wouldn't it be nice to keep all of these constants in one place? AuthHttpClient is a good spot to keep them. And it appears INTERNAL_ERROR is currently being duplicated both here and AuthHttpClient.

Reply:
Moved the remaining 2 error codes to `AuthHttpClient`